python/astropy.rst

```diff
@@ -28,6 +24,15 @@ The following packages can be used internally in packages if they do not leak in
 * :mod:`astropy.cosmology`
 * :mod:`astropy.visualization`
 
+For reading and writing files in FITS format, both :mod:`astropy.io.fits` (use of ``pyfits`` is now deprecated) and `fitsio`_ are allowed.
+While the latter usually surpassess the former in performance, developers must explicitly test with both these libraries and make a choice based on what works best for their use case.
```

Comment:
"surpasses".

I understand that there are differences in the functionality available between the two, which means that I'd rather say "developers must evaluate which works best for their use case" without requiring explicit testing.